goose.py

```diff
@@ -305,25 +312,30 @@ def digest(s: str) -> str:
 
 
 def get_db_migrations(conn) -> List[Migration]:
-    with conn:
-        # todo - namedtuple cursor
-        with conn.cursor() as cursor:
-            cursor.execute(
-                f"select migration_id, up_digest, up, down_digest, down from {migrations_table}"
+
```

Comment:
This line was the source of the psycopg2 problems, right?

Reply:
Correct.  I had to fix this before I could even test it with pytest-postgresql because that requires psycopg 3.0